### PR TITLE
[UI] Remove theme constants from twenty-front and use the ones exported from twenty-ui.

### PR DESCRIPTION
**Pull request by mabdullahabaid**
Some parts of the Frontend used theme constants exported from `modules/ui` while other parts used theme constants exported from `twenty-ui`.

This PR centralizes theme constants by removing them from `modules/ui` completely.

### CODE REVIEW

**Comment by charlesBochet** [comment]
@mabdullahabaid we will have a lot of conflicts with @lucasbordeau work that we are merging today, let's wait before merging this one!

**Comment by charlesBochet** [comment]
@mabdullahabaid Lucas' work has been merged, good to go, could you re-work this PR so we can merge it?

**Comment by mabdullahabaid** [comment]
@charlesBochet thank you for informing, on it.

**Comment by mabdullahabaid** [comment]
@charlesBochet reworked the PR to resolve conflicts, testing (and application) appears to be broken on main.

Screenshot 2024-05-25 at 2 32 48 PM

**Comment by charlesBochet** [comment]
I'm taking a look

**Comment by charlesBochet** [comment]
Here we go: https://github.com/twentyhq/twenty/pull/5573

**Comment by charlesBochet** [comment]
It's merged!

**Comment by mabdullahabaid** [comment]
Merged main onto my branch.

**Comment by mabdullahabaid** [comment]
Tests for modules and pages were green locally, expecting the same here.

**Review by charlesBochet — APPROVED** [review]
Thank you!

File: packages/twenty-ui/src/theme/constants/ThemeCommon.ts [hunk 1]
```diff
@@ -14,6 +15,20 @@ export const THEME_COMMON = {
   text: TEXT,
   blur: BLUR,
   animation: ANIMATION,
+  snackBar: {
```
Comment (charlesBochet) [inline packages/twenty-ui/src/theme/constants/ThemeCommon.ts:18]: I'm surprised these are not different between dark and light mode. We could also extract them into their own constant file

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**